src/central_control/scripts/central_control.py

```diff
@@ -25,21 +25,25 @@ def listen(self):
         rospy.Subscriber("braking", Float64, self.handle_breaking_data)
         rospy.Subscriber("throttling", Float64, self.handle_throttling_data)
         rospy.Subscriber("sign_detection", DetectionResult, self.handle_sign_recognition)
-        rospy.spin()
+
+        rate = rospy.Rate(30)
```

Comment:
This change had to be made otherwise there was a ton of bad callback errors

Reply:
Just include the rate.sleep in the while loop- see the review comment